src/main/java/com/bealetech/metrics/reporting/StatsdReporter.java

```diff
@@ -341,7 +341,12 @@ public void processTimer(String name, Timer timer) throws Exception {
     }
 
     public void processGauge(String name, Gauge<?> gauge) throws Exception {
-        sendObj(name + ".count", StatType.GAUGE, gauge.getValue());
+        //unfortunately the weakly typed Gauge can send some values that will break our statsd impl
+        if (gauge.getValue() instanceof Number) {
+            sendObj(name + ".count", StatType.GAUGE, gauge.getValue());
+        } else {
+            LOG.warn("Ignoring non numeric gauge " + name);
```

Comment:
since we have known circumstances that make this a regular occurrence - this could get kind of chatty... LOG.debug or trace instead?